sdk/search/Microsoft.Azure.Search.Service/src/Generated/Models/TextTranslationSkillLanguage.cs

```diff
@@ -0,0 +1,426 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Search.Models
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System.Runtime;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Defines values for TextTranslationSkillLanguage.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum TextTranslationSkillLanguage
```

Comment:
This is the autogenerated one right?  We need to switch to the custom one in my branch?

Reply:
Oh good catch, yes will do!

Reply:
I need to discuss with Bruce about this - let me keep the PR open until then

Reply:
Sure.  Its more so to be consistent with the other languages then anything else.

Reply:
Why do you need a custom one? Don't we use auto-generated enums for language in other situations?

Reply:
Spoke offline with Bruce - here's the deal

1. We shouldn't be using the customizations any more for the *SkillLanguage enums, because they are just that - not "extensible" per se
2. The reason why code is not generated for the other skill languages is because we explicitly exclude them in `generate.ps1`

I'll create a new PR to "fix" `generate.ps1` and will *block* getting this PR checked in before that's checked in

Reply:
It might be hard to generate a parallel PR, so I'll add a few more commits to this PR